Behaviors/GrayoutImageBehavior.cs

```diff
@@ -35,6 +55,9 @@ private static void OnImageIsEnabledChanged(object sender, DependencyPropertyCha
 
         private static void ToggleGrayOut(Image image)
         {
+            if (image.Source != null)
```

Comment:
This shouldn't have stayed in the code. Need to create bugfix branch and request another PR without this code.